crates/cdk/src/nuts/nut06.rs

```diff
@@ -170,6 +186,16 @@ impl MintInfo {
             ..self
         }
     }
+
+    /// Get time
+    pub fn time(&self) -> Option<u64> {
+        Some(
+            SystemTime::now()
+                .duration_since(UNIX_EPOCH)
+                .unwrap()
+                .as_secs(),
+        )
+    }
```

Comment:
We have this function in the util module we should be able to import it and reuse it here


https://github.com/cashubtc/cdk/blob/1cd80deaca28a63094931020bd90a26d9300998d/crates/cdk/src/util/mod.rs#L25